content_script.js

```diff
@@ -134,6 +89,91 @@ class HTMLSerializer {
   }
 
   /**
+   * Takes an html element, and populates this object's fields with the
+   * appropriate attribute names and values.
+   *
+   * @param {Element} element The Element to serialize.
+   * @private
+   */ 
+  processAttributes(element) {
```

Comment:
I think I'm a little mixed up about the terminology.  I thought that a class method is the same as a static method?  Additionally, I want this method to be an instance method, but I'm not sure what's different about this method than the other methods (most of which I also want to be instance methods).  Is the comment format incorrect?